configure.ac

```diff
@@ -190,8 +190,8 @@ AC_ARG_ENABLE([address-sanitizer], AS_HELP_STRING([--enable-address-sanitizer],
               [enabled AddressSanitizer support for detecting a wide variety of \
                memory allocation and deallocation errors]), \
               [AC_DEFINE(HAVE_ADDRESS_SANITIZER, 1, [enable AddressSanitizer])
```

Comment:
what about the other sanitizier cases?

Reply:
Does the bug occur with other sanitizers enabled?